eva-accession-release-automation/run_release_in_embassy/copy_accessioning_collections_to_embassy.py

```diff
@@ -43,18 +44,16 @@
 
 
 def mongo_data_copy_to_remote_host(local_forwarded_port, private_config_xml_file, assembly_accession,
-                                   collections_to_copy_map, dump_dir):
+                                   collections_to_copy_map, dump_dir, destination_db_name):
     mongo_params = parse_uri(get_mongo_uri_for_eva_profile("production", private_config_xml_file))
     # nodelist is in format: [(host1,port1), (host2,port2)]. Just choose one.
     # Mongo is smart enough to fallback to secondaries automatically.
     mongo_host = mongo_params["nodelist"][0][0]
     logger.info("Beginning data copy for assembly: " + assembly_accession)
     dump_output_dir = "{0}/dump_{1}".format(dump_dir, assembly_accession.replace(".", "_"))
-    destination_db_name = get_release_db_name_in_tempmongo_instance(assembly_accession)
 
-    # To be idempotent, clear source dump files and destination tempmongo database
+    # To be idempotent, clear source dump files
     shutil.rmtree(dump_output_dir, ignore_errors=True)
-    MongoClient(port=local_forwarded_port).drop_database(destination_db_name)
```

Comment:
This has been moved to [here](https://github.com/EBIvariation/eva-accession/pull/244/files#diff-40fa29c190a1786295e6c4d5f14e0ac8R103)